pages/blog/2022Q1-summary.md

```diff
@@ -0,0 +1,175 @@
+
+Alejandra and I will be mentors for interns for the next few months. Stay tuned to watch the progress.
+
+### AsyncAPI Mentorship
+
+Even though the Google Summer of Code (GSoC) application was declined, we did not want to give up. We knew many community members joined and started contributing because they counted on AsyncAPI being part of GSoC. 
+
+Well tl;dr our TSC just accepted an idea to start our own [AsyncAPI Mentorship](https://github.com/asyncapi/community/discussions/284) program :rocket:
+
+Now it is time until the 15th of May to discuss with mentors [in related ideas](https://github.com/issues?q=is%3Aopen+org%3Aasyncapi+label%3Agsoc) to understand the scope and clearly state you are a candidate for a given idea. Then we will ask TSC to vote and select 10 ideas that AsyncAPI Initiative should sponsor this year.
+
+## AsyncAPI Training
+
+ICYMI, we run an effort to create official AsyncAPI training videos. [Barbaño González](https://www.linkedin.com/in/barbano-gonzalez-moreno/) leads this topic and almost finished [scripts and storyboards](https://github.com/asyncapi/training/pulls) for the first set of videos.
+
+I do not think it is a secret if I say that great content is created only with great input, feedback, and review from people waiting for it. Please join Barbaño's efforts with any help as she is not doing it for herself but for the community.
+
+I recommend you [follow Barbaño's monthly updates](https://gist.github.com/Barbanio), especially if you want to participate.
+
+## Docs Feedback Form
+
+One of our missions is to have kick-ass documentation. Easy right? 
+
+Writing docs is easy, just like writing code. Well, we all know the devil is hidden in details. Details like gathering requirements, figuring out architecture, and making sure you have a well-motivated group of people working on it. And last but not least, make sure the community can easily share feedback on your work transparently.
+
+I wrote already about [Alejandra Quetzalli](https://twitter.com/QuetzalliAle) and the efforts to get AsyncAPI into GSoD. Getting a well-motivated group of people to work on docs, checked :white_check_mark:
+
+New docs information architecture is [almost here too](https://github.com/asyncapi/website/pull/601). I think we can say "checked" :white_check_mark: to this one too.
+
+Since this week, we can also say "checked" :white_check_mark: to the idea of getting feedback from the community.
+
+<TwitterTweetEmbed
+  tweetId='1518596333749391363'
+  options={{
+    width: 200,
+    maxWidth: 400
+  }}
+/>
+
+Each documentation page has a dedicated feedback form that one can fill in anonymously. As a result, feedback lands in [GitHub Discussion](https://github.com/asyncapi/community/discussions/340).
```

Comment:
```suggestion
Each documentation page has a dedicated feedback form that one can fill in anonymously. Your [docs feedback is injected anonymously into a Docs GitHub Discussion, as you can see here in this discussion link example](https://github.com/asyncapi/community/discussions/340).
```
The anchor text in this case did not adequately explain what the link it's pointing to is about. That's why I made a point to re-phrase this one a little more.

Reply:
This is not a good change. It will look bad 👇🏼 

![Screenshot 2022-04-27 at 10 03 23](https://user-images.githubusercontent.com/6995927/165471125-7b375c46-b7c7-4382-baec-95614bd39003.png)

my understanding of accessibility focused anchors is to make sure they describe what resource is hidden under a link and not "summarizes" the implementation. Also `anonymously` is repeated again

Reply:
I would go ahead and rephrase it and think about how you can still improve it :)

Your original implementation is still not good for accessibility so we still need to make it better ♥️

Reply:
> Each documentation page has a dedicated feedback form that one can fill in anonymously. Your feedback is passed into a docs-related GitHub Discussion, as you can [see here in this discussion example](https://github.com/asyncapi/community/discussions/340).

how about this?

Reply:
Each documentation page has a dedicated feedback form that one can fill in anonymously. The form injects the feedback into a [docs-related GitHub Discussion like this example](https://github.com/asyncapi/community/discussions/340).
 
___ 

Close, how about this @derberg? It's still unclear for screenreader users who land/skip through that link ("see here in this discussion example") because it does not clarify its kind of discussion example. I think that clarifying the discussion example is needed for this anchor text to be more accessible. :)

Reply:
like it! changed